spec/controllers/categories_controller_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
+
+RSpec.describe CategoriesController, type: :controller do
```

Comment:
Rails recommends to use requests specs ) https://medium.com/just-tech/rspec-controller-or-request-specs-d93ef563ef11

Reply:
@denys281 ok, I'll use requests specs for my future controllers